Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/Definitions/Input.cs

```diff
@@ -71,6 +72,33 @@ public class Input
     [DefaultValue(false)]
     public bool TypesOnly { get; set; }
 
+    /// <summary>
+    /// Specifies the encoding of the content.
+    /// </summary>
+    /// <example>ContentEncoding.UTF8</example>
+    public ContentEncoding ContentEncoding { get; set; }
+
+    /// <summary>
+    /// Enable BOM in UTF-8 encoding.
+    /// </summary>
+    /// <example>true</example>
+    [UIHint(nameof(ContentEncoding), "", ContentEncoding.UTF8)]
+    public bool EnableBom { get; set; }
+
+    /// <summary>
+    /// Content encoding as string.
+    /// </summary>
+    /// <example>windows-1251</example>
+    [UIHint(nameof(ContentEncoding), "", ContentEncoding.Other)]
+    public string ContentEncodingString { get; set; }
+
+    /// <summary>
+    /// Determine if only specified attributes should be returned.
+    /// This allows user to specify encoding and type for certain attributes.
+    /// </summary>
+    [DefaultValue(true)]
+    public bool SearchOnlySpecifiedAttributes { get; set; }
```

Comment:
I think that the Attributes parameter should now depend on this flag and be shown in Frends only when this is set to True?

Reply:
No, because the Task also allows for the user to specify certain attributes but still return all of them. This was done because if the user wants to return all attributes but knows that one of them is a byte array or a guide, then they can only specify that attribute and set it's type accordingly without having to specify all the other attributes.

Reply:
Got it, thanks